spec/services/providers/copy_to_recruitment_cycle_service_spec.rb

```diff
@@ -113,6 +114,12 @@
       expect(new_provider.organisation).to eq provider.organisation
     end
 
+    it "assigns the new provider to users" do
+      service.execute(provider: provider, new_recruitment_cycle: new_recruitment_cycle)
+
+      expect(new_provider.users).to eq provider.users
```

Comment:
Can you give some stats for how many providers and users will be rollovered for next cycle?

Reply:
As in this? https://github.com/DFE-Digital/teacher-training-api/blob/8ad1ccd7a7f613634be09b883d33879a608fdca4/app/services/providers/copy_to_recruitment_cycle_service.rb#L38-L42

Reply:
Or do you mean in the factory trait?

Reply:
yeah

Reply:
I guess you could have `expect(new_provider.users.count).to eq 4` but not sure what value that adds?